src/AWS.Deploy.Orchestration/CDK/CDKBootstrapTemplate.yaml

```diff
@@ -584,7 +584,7 @@ Resources:
       Type: String
       Name:
         Fn::Sub: /cdk-bootstrap/${Qualifier}/version
-      Value: "18"
+      Value: "19"
```

Comment:
If 207+208 is from us, did CDK not actually change anything 18 -> 19?

Reply:
Hmm, okay. Looks like they're made and then reverted two different changes recently: https://github.com/aws/aws-cdk/commits/main/packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml